stacks/container-apps/cms.prx.org.yml

```diff
@@ -379,6 +384,58 @@ Resources:
         - Key: "prx:cloudformation:stack-id"
           Value: !Ref AWS::StackId
       TaskDefinition: !Ref WorkerTaskDefinition
+
+  # memcached
+  CmsMemcacheSubnetGroup:
+    Type: AWS::ElastiCache::SubnetGroup
+    Properties:
+      Description: !Sub CMS ${EnvironmentType} memcache subnet group
+      SubnetIds:
+        - !Ref VPCSubnet1
+        - !Ref VPCSubnet2
+  CmsMemcacheSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      VpcId: !Ref VPC
+      GroupDescription: !Sub CMS ${EnvironmentType} memcached security group
+      SecurityGroupIngress:
+        - CidrIp: 0.0.0.0/0
+          IpProtocol: tcp
+          FromPort: 11211
+          ToPort: 11211
+        - CidrIpv6: ::/0
+          IpProtocol: tcp
+          FromPort: 11211
+          ToPort: 11211
+  CmsMemcacheCluster:
+    Type: AWS::ElastiCache::CacheCluster
+    Properties:
+      AutoMinorVersionUpgrade: true
```

Comment:
I usually leave this false, for apps that are may not be able to handle downtime.  But maybe CMS can?  What's the current setting?

Reply:
Even in prod, the Cfn template has this set to `true`. I can't really find what that maps to in console, so…yes?